docs/release-notes/known-issues.md

```diff
@@ -29,4 +29,29 @@ On 04/16/2025, a known issue was discovered that can cause a crash of the Event
 As a workaround, and to prevent crashes, Beats will ignore any filters provided when working with forwarded events on Windows Server 2025 until the issue is resolved.
 
 % **Resolved**
-% On [Month/Day/Year], this issue was resolved.
\ No newline at end of file
+% On [Month/Day/Year], this issue was resolved.
+
+:::
+
+:::{dropdown} Filebeat's Filestream input does not validate `clean_inactive`.
+**Applies to**: Filebeat < 9.2.0
+
+The Filestream input does not enforce the restrictions documented for
+the `clean_inactive` option, thus allowing configurations that can
+lead to data re-ingestion issues.
```

Comment:
Is there an issue and/or PR we can link to? Something like... `For more information, refer to [issue #](issue link)`.

Reply:
Yes and no.

There is the issue/task: https://github.com/elastic/beats/issues/45601, but I don't believe it is very helpful for users to read it.

The PR (https://github.com/elastic/beats/pull/46373) adding it does not contain much details about the issue, it's more about the fix.